lib/rubocop/cop/rspec/focus.rb

```diff
@@ -77,10 +77,10 @@ def on_send(node)
 
           focus_metadata(node) do |focus|
             add_offense(focus) do |corrector|
-              if focus.pair_type? || focus.str_type? || focus.sym_type?
```

Comment:
Focus can't be a str_type?, or can it?

Reply:
I don’t think so, no. But the code has been there forever 🤷🏼

Reply:
I love the gentle idea of dropping this condition by flipping else with elsif! 😄